### PR TITLE
Get user created markets loading in portfolio page

### LINKED ISSUE #2621: Integrate getters to populate portfolio page
- [x] get orders to display user's open orders
 - [x] get filled orders to show in portfolio page
 - [x] get user created markets
 - [x] get users positions data

### PR DESCRIPTION
**Pull request by bthaile**
fixes https://github.com/AugurProject/augur/issues/2621

getting users created markets is the last data need to populate Portfolio page. There is still some clean up for positions, getting user's position data is in another PR.

### CODE REVIEW

File: packages/augur-ui/src/modules/account/components/open-markets.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { SizeTypes } from "modules/types";
 import Styles from "modules/account/components/open-markets.styles.less";
 
 function filterComp(input: any, market: any) {
-  return market.description.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+  return market.description ? market.description.toLowerCase().indexOf(input.toLowerCase()) >= 0 : true;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/account/components/open-markets.tsx]: for edge cases when the market doesn't have a description, just include it in the list.

File: packages/augur-ui/src/modules/common/labels.tsx [hunk 2]
```diff
@@ -330,7 +330,7 @@ export class TextLabel extends React.Component<TextLabelProps, TextLabelState> {
         <label
           ref={ref => (this.labelRef = ref)}
           data-tip
-          data-for={`${keyId}-${text.replace(/\s+/g, "-")}`}
+          data-for={`${keyId}-${text ? text.replace(/\s+/g, "-") : ""}`}
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/labels.tsx]: Simple defensive code against empty label text value

**Review by phoebemirman — APPROVED** [review]

File: packages/augur-ui/src/modules/positions/selectors/positions-summary.ts [hunk 3]
```diff
@@ -69,4 +69,4 @@ export const positionSummary = memoize(
   },
 );
 
-const timesHundred = (value) => isNaN(value) ? "0" : createBigNumber(value).times(100);
+const timesHundred = (value) => isNaN(value) ? createBigNumber("0") : createBigNumber(value).times(100);
```
Comment (justinbarry) [inline packages/augur-ui/src/modules/positions/selectors/positions-summary.ts:72]: @bthaile Is it possible `value` could be a bignumber here?

Comment (bthaile) [reply packages/augur-ui/src/modules/positions/selectors/positions-summary.ts:72]: nope it's not because of our Formatting helpers

**Review by justinbarry — APPROVED** [review]
Just one comment. Otherwise it lgtm.

**Pull request merged**